packages/ts/react-signals/test/Signals.spec.tsx

```diff
@@ -59,5 +59,16 @@ describe('@vaadin/hilla-react-signals', () => {
       await nextFrame();
       expect(result.container.textContent).to.equal('Value is 42');
     });
+
+    it('should set local value when signal value is updated', () => {
+      const numberSignal = new NumberSignal(publishSpy);
+      expect(numberSignal.value).to.equal(undefined);
+      numberSignal.value = 42;
+      expect(numberSignal.value).to.equal(42);
```

Comment:
Should maybe also test that an `effect(() => doSomething(numberSignal.value))` is run already before the server has confirmed the change